web/backend-fastapi/app/controllers/feedback.py

```diff
@@ -1,35 +1,12 @@
 from fastapi import APIRouter, Form
-from app.dependencies import get_user_info
-from langchain_community.embeddings import HuggingFaceEmbeddings
-from app.models import neo4j, trulens, rag
+from app.models import trulens
 import json
 import logging
 
 router = APIRouter()
 
 kg = None
 tru = None
-embeddings = HuggingFaceEmbeddings(model_name="all-MiniLM-L6-v2")
-
-@router.post("/submit/")
```

Comment:
are we sure this is not breaking embedding adaptors? please double check

Reply:
I don't believe so. These embeddings were never used in the feedback controller, and the frontend never calls this route. Perhaps it's being called from somewhere outside of the frontend?